From 3b7c1a6f866f9dc241ad7222ae2f224236a453cc Mon Sep 17 00:00:00 2001
From: Alejandro Martinez Ruiz <alex@nowcomputing.com>
Date: Thu, 24 Jan 2013 20:56:25 +0100
Subject: [PATCH 03/13] Revert "f2fs: add remap_pages as
 generic_file_remap_pages"

This reverts commit a077ffb369af6b981113cda04d61784cbc8cd09d.

The original commit was meant to include support for a feature not
present in 3.2 kernels.

Signed-off-by: Alejandro Martinez Ruiz <alex@nowcomputing.com>
---
 fs/f2fs/file.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/fs/f2fs/file.c b/fs/f2fs/file.c
index 8628f5c..2ccf9f2 100644
--- a/fs/f2fs/file.c
+++ b/fs/f2fs/file.c
@@ -105,9 +105,8 @@ out:
 }
 
 static const struct vm_operations_struct f2fs_file_vm_ops = {
-	.fault		= filemap_fault,
-	.page_mkwrite	= f2fs_vm_page_mkwrite,
-	.remap_pages	= generic_file_remap_pages,
+	.fault        = filemap_fault,
+	.page_mkwrite = f2fs_vm_page_mkwrite,
 };
 
 static int need_to_sync_dir(struct f2fs_sb_info *sbi, struct inode *inode)
-- 
1.8.1.2.422.g08c0e7f

